packages/twilio-client/mockClient/getMockClient.ts

```diff
@@ -0,0 +1,168 @@
+import RestException from 'twilio/lib/base/RestException';
+import { chatMessageList } from './chatMessageList';
+
+export const getMockClient = () => {
+  return {
+    chat: {
+      v2: {
+        services: () => ({
+          channels: {
+            get: () => ({
+              messages: {
+                list: () => chatMessageList,
+              },
+            }),
+          },
+          users: {
+            get: (identity: string) => {
+              let userData: {
+                sid: string;
+                accountSid: string;
+                serviceSid: string;
+                attributes: string;
+                friendlyName: string;
+                roleSid: string;
+                identity: string;
+                isOnline: boolean;
+                isNotifiable: boolean;
+                dateCreated: Date;
+                dateUpdated: Date;
+                joinedChannelsCount: number;
+                links: {
+                  user_channels: string;
+                  user_bindings: string;
+                };
+                url: string;
+              };
+              switch (identity) {
+                case 'mockServiceUserIdentity':
+                  userData = {
```

Comment:
These and the below role related data are real-ish responses that have been stripped of some specific data that I thought may be best kept out of the codebase. It should give you an idea of what the real life data being added to the participants property of the transcript will look like.